.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: bpfci
+
+on:
+  pull_request:
+
+jobs:
+  VM_Test:
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_NAME: 'libbpf'
+      AUTHOR_EMAIL: "$(git log -1 --pretty=\"%aE\")"
+      KERNEL: LATEST
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: install
+        run: for f in aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils; do sudo apt-get install $f ; done
```

Comment:
you can probably do just `sudo apt-get install aptitude qemu-kvm zstd binutils-dev elfutils libcap-dev libelf-dev libdw-dev python3-docutils` without for loop